basics/data-import/pinot-stream-ingestion/README.md

```diff
@@ -101,9 +101,20 @@ The ingestion configuration (`ingestionConfig`) specifies how to ingest streamin
-| `realtime.segment.flush.threshold.rows`               | The maximum number of rows to consume before persisting the consuming segment. If this value is set to 0, the configuration looks to `realtime.segment.flush.threshold.segment.size` below. | Default is 5,000,000                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `realtime.segment.flush.threshold.rows`               | The maximum number of rows to consume before persisting the consuming segment. If this value is set to 0, the configuration looks to `realtime.segment.flush.threshold.segment.size` below. See note below this table for more information. | Default is 5,000,000                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
 | `realtime.segment.flush.threshold.segment.size`       | Desired size of the completed segments. This value is used when `realtime.segment.flush.threshold.rows` is set to 0.                                                                        | String, such as `150M` or `1.1G`., etc. Default is `200M` (200 megabytes). You can also specify additional configurations for the consumer directly into `streamConfigMaps`. For example, for Kafka streams, add any of the configs described in [Kafka configuration page](https://kafka.apache.org/documentation/#consumerconfigs) to pass them directly to the Kafka consumer.                                                                                                 |
 
+{% hint style="info" %}
+When specifying `realtime.segment.flush.threshold.rows`, the actual number of rows per segment is computed using the following formula:
+
+```
+realtime.segment.flush.threshold.rows / partitionsConsumedByServer
+```
+
+This means that if we set `realtime.segment.flush.threshold.rows=1000` and each server consumes 10 partitions, the rows per segment will be `1000/10 = 100`.
```

Comment:
```suggestion
The number of rows per segment is computed using the following formula:
`realtime.segment.flush.threshold.rows /partitionsConsumedByServer`
For example, if you set `realtime.segment.flush.threshold.rows=1000` and each server consumes 10 partitions, the rows per segment is `1000/10 = 100`.
```

Reply:
I might add this note to the table itself

Reply:
I almost did that. My fear was that the cell it would go in is already quite large and I worried it would make the table too hard to read. I'll give it further thought before I move or don't move this.